test/unit/region.spec.js

```diff
@@ -1104,12 +1042,14 @@ describe('region', function() {
     let region;
 
     beforeEach(function() {
-      itemView = new View();
-      itemView.render = this.sinon.stub();
-      itemView.addRegions({
-        MyRegion: '#region',
-        anotherRegion: '#region2'
+      itemView = new View({
```

Comment:
this test broke because it was `noop`ing render which is now what builds the regions it was testing